src/v2/view-builder/views/shared/ChallengeAuthenticatorDataView.js

```diff
@@ -1,17 +1,18 @@
+// Common view for OV push and custom push.
```

Comment:
similar here about the name of the file. Something that say what this view is for is preferred. See my comment in the other view